Lib/glyphsLib/builder/user_data.py

```diff
@@ -114,10 +113,7 @@ def to_glyphs_master_user_data(self, ufo, master):
         ufo_data = {}
         for os_filename in ufo.data.fileNames:
             filename = posixpath.join(*os_filename.split(os.path.sep))
-            data_bytes = base64.b64encode(ufo.data[os_filename])
-            # FIXME: (jany) The `decode` is here because putting bytes in
-            # userData doesn't work in Python 3. (comes out as `"b'stuff'"`)
```

Comment:
> FIXME: (jany) The `decode` is here because putting bytes in userData doesn't work in Python 3. (comes out as `"b'stuff'"`)

the reason of course was that the current Writer did not explicitly handle binary data as hex strings (unlike the Parser which Nikolaus and Jany had extended to support parsing these). It was simply calling `unicode(value)` and then escaping the quotes.